packages/web/src/App.tsx

```diff
@@ -22,7 +23,8 @@ const App = () => (
       <ConnectedRouter history={history}>
         <Switch>
           <Route path="/signin" component={SignIn} />
-          <Route path="/signout" component={SignOut} />
+          <Route path="/my-shelfs" component={MyFlowerShelfs} />
```

Comment:
I'd suggest path simple 'shelfs' and name can also be "My Shelfs"

Reply:
and maybe place it in the bottom, not between signin signout

Reply:
Fixed as agreed with @belgithub